lib/helpers/contentfulEntry.js

```diff
@@ -223,8 +223,9 @@ function isLegacyBroadcast(contentfulEntry) {
  * @param {Object} contentfulEntry
  * @return {Boolean}
  */
-function isMessage(contentfulEntry) {
-  return contentful.isContentType(contentfulEntry, config.contentTypes.message.type);
+function isFaqAnswer(contentfulEntry) {
+  return [config.contentTypes.message.type, config.contentTypes.faqAnswer.type]
```

Comment:
Just a nit, it would be a bit more readable this way:
```
  return [
    config.contentTypes.message.type,
    config.contentTypes.faqAnswer.type,
    ]
```

Reply:
Agreed -- I'm going to leave for now as this code should hopefully be toast soon 🍞 . It's been great having `prettier` integrated over in our [https://github.com/DoSomething/graphql](GraphQL) linting to catch and fix things like these... definitely would be worth adding into Conversations!